glitter/templates/glitter/include/startbody.html

```diff
@@ -12,6 +12,9 @@
                 </li>
                 {% if glitter.has_add_permission %}
                     <form action="{% url glitter.opts|admin_urlname:'add' %}">
+                        {% if glitter.obj.is_root_node and glitter.obj.get_absolute_url != "/" %}
```

Comment:
Could you remove the `glitter.obj.is_root_node` - child nodes can have children added to them too.

The rest is fine.